src/ui/components/UserRating/index.js

```diff
@@ -12,6 +12,7 @@ import type { UserReviewType } from 'amo/actions/reviews';
 type Props = {|
   className?: string,
   isOwner?: boolean,
+  loading?: boolean,
```

Comment:
Wouldn't `loading` just be the same as `!readOnly && review === undefined`? Or do you need this for testing purposes?

Reply:
The `RatingManager` needs to present stars that the user can select to rate the component. In this case, `rating==undefined` because no rating has been set. My intention was to let `RatingManager` control the loading state but let me double check to see I still need to do that.